setup.py

```diff
@@ -53,15 +53,15 @@
         'packaging',
         'pandas!=2.2.0,!=2.2.1,!=2.2.2',  # ESMValCore PR2305
         'progressbar2',
-        'psyplot',
-        'psy-maps',
-        'psy-reg',
-        'psy-simple',
+        'psyplot>=1.5.0',
+        'psy-maps>=1.5.0',
+        # 'psy-reg',  # not yet at 1.5.0 not yet py312
```

Comment:
Uncomment?

Reply:
pffft what a n00b I am :grin:

Reply:
done in a859921f202d3cce55c1c9d572bee83724761ad9 and also added comment why >=1.5 for all psy pkgs